index.d.ts

```diff
@@ -1 +1,16 @@
-export declare function isMobile(ua?: string|object, opts?: object): boolean;
```

Comment:
is this going to be a breaking change?

Reply:
Sure! Here described two optional parameters, but function expecting an optional object, with two optional parameter one of which is node's request object.
Long things short - mine is the poper one (and declared in package.json - it is necessary to make typings work for external modules) %)

Reply:
@egorovsa you had initially created the typedef, are you cool with this?